src/uiTest/resources/project.rhino.js

```diff
@@ -282,18 +282,28 @@ function Project(underlyingProject) {
   };
 
   this.moveMouseToTheMiddleAndWait = function (secondsToWait) {
-    const ideFrame = getComponentByClass('com.intellij.openapi.wm.impl.IdeFrameImpl');
+    const ideFrame = getIdeFrame();
     robot.moveMouse(ideFrame);
-    for (var i = 0; i < 10 * secondsToWait; i++) {
+    for (let i = 0; i < 10 * secondsToWait; i++) {
       sleep();
     }
   };
 
   const clickMouseInTheMiddle = function () {
-    const ideFrame = getComponentByClass('com.intellij.openapi.wm.impl.IdeFrameImpl');
+    const ideFrame = getIdeFrame();
     robot.click(ideFrame);
   };
 
+  const getIdeFrame = function() {
+    return getComponentByClass(
+        /* className */ 'com.intellij.openapi.wm.impl.IdeFrameImpl',
+        /* text */ 'machete-sandbox-worktree',
```

Comment:
Would it make sense to just squeeze the two params into a `predicate` in `getComponentByClass`?
Like 
```
/* predicate */ component => 'machete-sandbox-worktree'.equals(component.getTitle())
```

Reply:
good idea